scripts/psycorona_modelling.R

```diff
@@ -69,6 +73,77 @@ preds        <- names(df_analyse)[-(which(names(df_analyse) %in% outcome_vars))]
 # Explore outcome variable
 hist(df_analyse$c19perbeh)  # people tend to self-serve
 
+
+########## CARE MODEL COMPARING SETUP ########## 
+
+# make the CV split
+cv_split <- trainControl(method="cv", number=10, verboseIter=TRUE)
+# Train the LASSO 
+lambda <- seq(0.0001,0.01,by = 0.0001)
+# set seed
+set.seed(100)
+caret_lasso <- train(c19perbeh~., data=df_analyse, 
+                     method="glmnet", 
+                     trControl=cv_split, 
+                     family = "gaussian", 
+                     tuneGrid = expand.grid(alpha = 1, lambda = lambda), 
```

Comment:
I think the alpha argument can be left out, because it's set to 1 by default

Reply:
I think that's indeed the case if we run a default glmnet But when I run it with the caret train function then it fits a sequence of different alphas between 0 and 1